package.json

```diff
@@ -40,7 +40,7 @@
     "mini-css-extract-plugin": "2.7.6",
     "moment": "2.30.1",
     "node-bourbon": "^4.2.3",
-    "openmct": "nasa/openmct#omm-r5.3.1",
+    "openmct": "github:nasa/openmct#omm-release/5.4.0",
```

Comment:
Curious what the difference in adding github is?

Reply:
Yeah, npm kept updating it to that because it's now standard practice to be specific about the source. So I just added it so I didn't have to keep removing it.